datastore/load-test/schedule_write.py

```diff
@@ -0,0 +1,101 @@
+import math
+import os
+import random
+import time
+import uuid
+from collections import namedtuple
+from datetime import datetime
+from datetime import UTC
+
+import datastore_pb2 as dstore
+import datastore_pb2_grpc as dstore_grpc
+import grpc
+from apscheduler.executors.pool import ThreadPoolExecutor
+from apscheduler.schedulers.blocking import BlockingScheduler
+from apscheduler.triggers.cron import CronTrigger
+from google.protobuf.timestamp_pb2 import Timestamp
+from variables import variable_info
+
+# from apscheduler.executors.pool import ProcessPoolExecutor
+
+
+# One channel & client per process (not per thread!)
+# TODO: Does this affect load on the database? Seems load here is lower then in Rosina's code
+channel = grpc.insecure_channel(f"{os.getenv('DSHOST', 'localhost')}:{os.getenv('DSPORT', '50050')}")
+client = dstore_grpc.DatastoreStub(channel=channel)
```

Comment:
Can a single channel be a bottleneck for the load test? I am wondering if this could be the case as I read that a channel has a limit on the number of concurrent streams. When the limit is reached, the RPCs are queued.

https://grpc.io/docs/guides/performance/

Reply:
Added as a comment, will look at it some time later.